tekton/publish-customtask.yaml

```diff
@@ -140,9 +140,8 @@ spec:
         IMAGES_PATH=${IMAGES_PATH}/$(params.subfolder)
       fi
 
-      for d in cmd/*
-      do
-        IMAGES="${IMAGES} ${IMAGES_PATH}/${d}:$(params.versionTag)"
+      for d in ${PROJECT_ROOT}/cmd/*; do
```

Comment:
I've tested this out on my cluster and it works. The issue was that if there is a `cmd` folder in the current directory, `cmd/*` is interpreted as a list, but otherwise it's interpreted as a literal string.